tests/test_forge_data.py

```diff
@@ -0,0 +1,15 @@
+raw_response = {
```

Comment:
I don't see any usages of this variable. Remove this file, please.

Reply:
Fixed in d74df88